subsys/bluetooth/audio/vocs.c

```diff
@@ -46,8 +46,23 @@ static void location_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value
 	BT_DBG("value 0x%04x", value);
 }
 
+static ssize_t read_location(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			     void *buf, uint16_t len, uint16_t offset)
+{
+	struct bt_vocs *inst = attr->user_data;
+
+	BT_DBG("0x%02x", inst->srv.location);
+	return bt_gatt_attr_read(conn, attr, buf, len, offset, &inst->srv.location,
+				 sizeof(inst->srv.location));
+}
+
+static void output_desc_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value)
+{
+	BT_DBG("value 0x%04x", value);
+}
 #endif /* CONFIG_BT_VOCS */
 
+#if defined(CONFIG_BT_VOCS) || defined(CONFIG_BT_VOCS_CLIENT)
```

Comment:
`vocs.c` should never be included in the compiling if neither of these are enabled, and this is thus an unnecessary check.